src/app/rundown-overview/components/rundown-overview/rundown-overview.component.html

```diff
@@ -1,5 +1,5 @@
 <sofie-header></sofie-header>
-<sofie-page i18n-title title="rundown-overview.rundown-overview.title">
+<sofie-page>
```

Comment:
Why is the title removed?
We still want to have the "title" that says `Rundowns` on the overview page.

I see that the unwanted "tooltip" is now gone from the RundownOverview page. But it is still there on the Settings page. So this does not solve SOF-1843.

It looks like the issue comes from `i18n-title` where `title` seems to be a keyword that applies `i18n` applies to a lot of things. Changing the input variable `title` on `SofiePage` to a different name fixes the bug.

Sidenote: In solution you have removed the usage of a translation key, but you haven't removed the translation from the translation files which leaves us with unused translations.

Reply:
@LindvedKrvang Sorry, I didn't understand the assignment properly. I think now the solution is the way you want it.